asset/src/kafka_sender_api/api.ts

```diff
@@ -30,7 +30,6 @@ export default class KafkaSenderApi extends APIFactory<KafkaRouteSender, KafkaSe
         if (isNil(config.required_acks) || !isNumber(config.required_acks)) throw new Error(`Parameter required_acks must be provided and be of type number, got ${getTypeOf(config.required_acks)}`);
         if (isNil(config.logger)) throw new Error(`Parameter logger must be provided and be of type Logger, got ${getTypeOf(config.logger)}`);
 
-        config.bufferSize = config.size * 5;
```

Comment:
By removing that, does this break https://github.com/terascope/kafka-assets/blob/2d3587693d55ad4a89d248579aa5f18edb1f3a09/asset/src/_kafka_clients/producer-client.ts#L20

Reply:
maybe a million is excessive - I figured that it was 5 x size, I bumped it to match high end of a slice size.  I tested out a variety of different values looking at slice times and didn't see a difference in times so long as the job could run.

Good question about the `this._bufferSize`.  I didn't test how these changes affect a single cluster and topic.  I'll run some test jobs and see.

Reply:
couple of points:
  * ended up setting config.bufferSize = config.max_buffer_size
  * default max_buffer_size is 100,000 - I can't tell a difference when setting it to 100K or 1M.
  * also spent some time editing the kafka_sender_api docs -
  * the default size setting in the routed_sender needs to be changed - even with the increased value for the max_buffer_size if the slice size is too small I get a different kafka error `Failure when performing produce, caused by error: timed out, code: -1, desc: "Unknown broker error"` - setting the size to 10k or above eliminated that error.
  * this is tricky when using the routed_sender and the kafka_api, because the routed_sender size overwrites the kafka_sender size setting, which isn't obvious
  * the routed_sender is also pretty slow, like 30s processing time per slice and it's only processing about 10k - 20k records a slice, but that is another issue